foundations/foundation-ssl/src/test/java/io/servicecomb/foundation/ssl/SSLOptionTest.java

```diff
@@ -230,6 +230,46 @@ public void testSSLOptionYamlOption2() throws Exception {
     System.clearProperty("ssl.protocols");
   }
 
+  @Test
+  public void testSSLOptionYamlOptionWithProperyFalse() throws Exception {
+    System.setProperty("ssl.authPeer", "false");
+    DynamicConfiguration configFromYamlFile =
```

Comment:
The below code is duplicated code, it's better to extract a method for setting up the configuration from system property.

Reply:
For unit test case, I think it quite important to make each test have only one logic (that is to say, code complexity is one) and code duplication is not a concern. So I used to copy code for new test cases.